src/utils/config.ts

```diff
@@ -40,6 +41,10 @@ export const SpidParams = t.intersection([
 
     ORG_NAME: NonEmptyString,
     ORG_URL: NonEmptyString,
+
+    JWT_TOKEN_EXPIRATION: NonNegativeInteger,
+    JWT_TOKEN_ISSUER: NonEmptyString,
+    JWT_TOKEN_PRIVATE_KEY: NonEmptyString,
```

Comment:
These properties can be removed from here because are they defined in `JWTParams`
```suggestion
```